src/physics/initial_conditions/initial_condition.cpp

```diff
@@ -139,8 +139,24 @@ InitialConditionFactory<dim,real>::create_InitialConditionFunction(
     return nullptr;
 }
 
+// ========================================================
+// ZERO INITIAL CONDITION
+// ========================================================
+template<int dim, typename real>
+InitialConditionFunction_Zero<dim, real> 
+::InitialConditionFunction_Zero(const unsigned int nstate) 
+    : dealii::Function<dim,real>(nstate)
+{ }
```

Comment:
Lets move this empty constructor to the header file since this might become quite cluttered eventually

Reply:
Done.